roles/kubernetes-apps/ansible/tasks/dashboard.yml

```diff
@@ -1,10 +1,20 @@
 ---
+- name: Kubernetes Apps | Delete old kubernetes-dashboard resources
+  kube:
+    name: "kubernetes-dashboard"
+    kubectl: "{{bin_dir}}/kubectl"
+    resource: "{{ item }}"
+    state: absent
+  with_items: ['ClusterRoleBinding']
```

Comment:
this is not critical, but can you please write all lists more readable like:
with_items:
    - ClusterRoleBinding
this way it's more readable when overlooking code, and more easy to add/remove more items if needed anywhere.

Reply:
I agree that's better, it's usually my preferred syntax. I was just sticking with the convention I saw used elsewhere in this role for dns cleanup.